iron-doc-hide-bar.html

```diff
@@ -37,7 +37,7 @@
     <a href="" on-tap="_toggle">
       <span hidden$="[[visible]]">Show</span>
       <span hidden$="[[!visible]]">Hide</span>
-      <slot></slot>
+      &#8203;<slot></slot>
```

Comment:
Whitespace was being collapsed by html-minifier (which is what polymer cli uses). Throwing in zero width spaces is a gross hack that forces it to not collapse as aggressively in these cases. I think maybe we should be using the `conservativeCollapse` option (https://www.npmjs.com/package/html-minifier#options-quick-reference), because it's pretty frustrating when the dev version renders differently to the built version.